INSTRUCTION.md

```diff
@@ -0,0 +1,64 @@
+# Інструкція по запуску та зупинці контейнерів
+Ця інструкція допоможе вам налаштувати і запустити контейнеризований додаток Django разом з MySQL за допомогою Docker та Docker Compose.
+
+1. Підготовка до запуску
+1.1 Клонування репозиторію Спочатку клонувати репозиторій на локальний комп'ютер:
+
+# git clone https://github.com/BlackYOSHl/devops_todolist_docker_core_task_3_docker_compose.git
+cd devops_todolist_docker_core_task_3_docker_compose
+
+1.2 Переконатися, що у вас встановлені Docker та Docker Compose
+
+Docker
+Docker Compose
+
+1.3 Перевірка Docker та Docker Compose Перевірте версії:
+
+# docker --version
+
+# docker-compose --version
+
+2. Запуск контейнерів
+
+2.1 Будування і запуск контейнерів Для побудови і запуску контейнерів виконайте команду:
+
+# docker-compose up --build
+
+Команда --build буде пересобирати образи, якщо є зміни в Dockerfile чи інших файлах.
+Docker Compose автоматично завантажить необхідні образи, збудує контейнери та запустить їх.
+
+2.2 Перевірка статусу контейнерів Для перевірки, чи працюють контейнери, використовуйте команду:
+
+# docker-compose ps
+
+Вона покаже інформацію про запущені контейнери.
+
+2.3 Перевірка додатку Після запуску контейнерів перейдіть до браузера і відкрийте http://localhost:8080. Ви повинні побачити веб-додаток Django.
+
+3. Виконання міграцій бази даних вручну (якщо необхідно)
+Міграції бази даних виконуються автоматично під час запуску контейнера. Якщо з якоїсь причини потрібно виконати міграції вручну, скористайтесь наступною командою:
```

Comment:
It's mentioned that database migrations are performed automatically during container startup. Ensure that this is accurately reflected in the Dockerfile and entrypoint script. If manual intervention is required, this should be clarified here.